macros/PGbasicmacros.pl

```diff
@@ -2919,7 +2919,8 @@ =head2 Macros for displaying static images
 
 #   More advanced macros
 sub image {
-	my $image_ref  = shift;
+	my $image_ref = shift;
+	$image_ref = main::insertGraph($image_ref) if (ref($image_ref) eq 'WWPlot' or ref($image_ref) eq 'TikZImage');
```

Comment:
There is no need for the `main::` namespace.  We are already in the `main` namespace here, and not in another package namespace.

Also, it is a good idea to get into the habit of using `||`, `&&`, and `!` instead of `or`, `and`, and `not` in logic statements.  It is a good rule of thumb to only use the word versions in flow control.  The word versions can cause issues if you aren't careful due to their low precedence.  The operators `||` and `or` are not equivalent.

Of course, another change to the image subroutine means another change to the POD usage is needed!  It would probably be enough to add a comment below the `Usage:` examples about the types of objects that the `$image` parameter may be.